core/templates/dev/head/pages/error/Error.js

```diff
@@ -22,5 +22,15 @@ oppia.controller('Error', [
       $scope, UrlInterpolationService) {
     $scope.oopsMintImgUrl = UrlInterpolationService.getStaticImageUrl(
       '/general/oops_mint.png');
+
+    $scope.ErrorCode = GLOBALS.errorCode;
+    $scope.ErrorCodeIs400 = function () {
```

Comment:
Why are you only treating the 400 case and not the others?

Also, in the html page I recommend doing `<div ng-if="errorCode === '400'">` or similar. I don't think you need a separate function for this.

Reply:
I am only treating the 400 case because in the issue #3950 it is only asked to change the occurrence of 400. Do you want me to change other cases?

Reply:
@Nix07, Oh sorry for not specifying it clearly, but by `{% if code == 400 %}` I meant the whole block till you see an `{% endif %}`, so that would include all the elif parts too!